### PR TITLE
Restore `toggle-files-button` in "Repository refresh" beta

### PR DESCRIPTION
**Pull request by FloEdelmann**
Relates to #3081.

With feature preview enabled:

Screenshot_2020-06-06_14-12-13

With feature preview disabled:

Screenshot_2020-06-06_14-12-38

### CODE REVIEW

File: source/features/toggle-files-button.css [hunk 1]
```diff
@@ -12,7 +12,8 @@
 	transform: rotate(0.5turn);
 }
 
-.rgh-files-hidden .files {
+.rgh-files-hidden .files,
```
Comment (fregante) [inline source/features/toggle-files-button.css:15]: Mark the old style with this code so in the future we know what to drop
```suggestion
.rgh-files-hidden .files, /* Pre "Repository refresh" layout */
```

File: source/features/toggle-files-button.css [hunk 2]
```diff
@@ -12,7 +12,8 @@
 	transform: rotate(0.5turn);
```
Comment (fregante) [inline source/features/toggle-files-button.css:12]: I think we got this wrong. "Down" means "open" while "up" means closed. You can see this in this very page for each collapsable file.

Do you mind dropping `:not()` from this selector in this PR? It's a minor change

Comment (FloEdelmann) [reply source/features/toggle-files-button.css:12]: In PR Files tabs, "down" means *open* while **"right"** means *closed*. So I think this for this feature, it should be "down" for open and "left" for *closed*.

Comment (fregante) [reply source/features/toggle-files-button.css:12]: You're right. Perhaps we should just use octicon-fold/octicon-unfold instead. In another PR

**Pull request merged**